context.go

```diff
@@ -0,0 +1,69 @@
+package api2go
+
+import (
+	"golang.org/x/net/context"
+	"time"
+)
+
+// APIContext api2go context for handlers, nil implementations related to Deadline and Done.
+type APIContext struct {
```

Comment:
an interface for that could be added and used as type everywhere, in that case the enduser can use this APIContext or implement his own if he wants too.